core/storage/user/gae_models_test.py

```diff
@@ -1100,6 +1102,8 @@ class ExplorationUserDataModelTest(test_utils.GenericTestBase):
 
     NONEXISTENT_USER_ID = 'id_x'
     DATETIME_OBJECT = datetime.datetime.strptime('2016-02-16', '%Y-%m-%d')
+    DATETIME_EPOCH = time.mktime(
+        datetime.datetime.strptime('2016-02-16', '%Y-%m-%d').timetuple())
```

Comment:
Why is this different from the other _EPOCH fields (they don't use strptime) but named in a similar way?

Reply:
Changed this to use the constant. This strptime was already there in these tests prior to me adding functionality, so I didn't want to change someone else's code. But my EPOCHs no longer use the strptime